app/src/main/java/ingsw/group1/findmyphone/alarm/ParserAlarmMessage.java

```diff
@@ -0,0 +1,36 @@
+package ingsw.group1.findmyphone.alarm;
+
+import android.location.Location;
+
+/**
+ * Manager that provides composition of message for alarm request and response
+ * Message request is composed as:
+ *  ALARM_REQUEST
```

Comment:
I believe something might be missing here

Reply:
For the actual composition of alarm request, the message is only 'ALARM_REQUEST'. We have to talk and decide about it.